src/Tabs/TabGroup.js

```diff
@@ -0,0 +1,97 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import { Tab } from './Tab';
+import { TabContent } from './TabContent';
+import React, { Component } from 'react';
+
+export class TabGroup extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedId: props.selectedId
+        };
+    }
+
+    // set selected tab
+    handleTabSelection = (event, id) => {
+        event.preventDefault();
+        this.setState({
+            selectedId: id
+        });
+    };
+
+    // create tab list
+    renderTabs = () => {
+        return React.Children.map(this.props.children, (child) => {
+            return (
+                <li {...this.props.tabLinkProps}
+                    className='fd-tabs__item'
+                    key={child.props.id}>
+                    <Tab {...child.props} onClick={this.handleTabSelection}
+                        selected={this.state.selectedId === child.props.id} />
+                </li >);
+        });
+    };
+
+    // create content to show below tab list
+    renderContent = () => {
+        return React.Children.map(this.props.children, (child) => {
+            return (
+                <TabContent
+                    {...this.props.tabContentProps}
+                    selected={this.state.selectedId === child.props.id}>
+                    {child.props.children}
+                </TabContent>);
```

Comment:
This is an interesting approach (using an internal component) to help encapsulate the details for tab content.  Per @bcullman's comment, we should just take a couple other steps to make sure this stays "private".